BMM.UI.Android/Application/Fragments/PlayerFragment.cs

```diff
@@ -410,16 +413,32 @@ private void UpdateTitleSize()
             {
                 if (Context == null)
                     return;
-                
+
+                SetWatchBccButtonPosition();
```

Comment:
BccMedia

Reply:
If you want a shorter name, you could call it WatchButton instead of WatchBccMediaButton